README.md

```diff
@@ -62,7 +68,7 @@ optional arguments:
                         https://github.com/<owner>/<repository>
   --upstream-branch UPSTREAM_BRANCH
                         branch of the upstream repository
-  --skip-keys SKIP_KEYS
+  --skip-keys SKIP_KEYS [SKIP_KEYS ...]
```

Comment:
Should be `SKIP_KEY [SKIP_KEY ...]` (and line 180, too) -- but please change in an separate commit.

Reply:
You mean to change the parser? Because this is copy pasted output from the parser.

Reply:
Yes.

Reply:
In other words, don't correct the `README.md` for this as part of this commit. Instead, create another commit (could be part of this PR) that corrects the usage message by passing the `formatter_class=argparse.MetavarTypeHelpFormatter` argument and the specifying `type=SKIP_KEY` argument to `.add_argument()`  for `--skip-keys` (see example for it at https://docs.python.org/3/library/argparse.html#argparse.MetavarTypeHelpFormatter).

Reply:
And if you wanted to create a third comment that changes the description to `get_parser()` in `bitbake/run.py`  to `Generate OpenEmbedded recipes for ROS packages`, that would be cool.

Reply:
> type=SKIP_KEY

did you mean metavar='SKIP_KEY'; type=SKIP_KEY doesn't seem to work and formatter_class=argparse.MetavarTypeHelpFormatter causes get_parser to fail with:
```
Traceback (most recent call last):
  File "/usr/local/bin/superflore-gen-oe-recipes", line 11, in <module>
    load_entry_point('superflore==0.3.1+10.gb9eb242.dirty', 'console_scripts', 'superflore-gen-oe-recipes')()
  File "/usr/local/lib/python3.6/dist-packages/superflore-0.3.1+10.gb9eb242.dirty-py3.6.egg/superflore/generators/bitbake/run.py", line 54, in main
  File "/usr/lib/python3.6/argparse.py", line 1743, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/usr/lib/python3.6/argparse.py", line 1775, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/usr/lib/python3.6/argparse.py", line 1981, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/usr/lib/python3.6/argparse.py", line 1921, in consume_optional
    take_action(action, args, option_string)
  File "/usr/lib/python3.6/argparse.py", line 1849, in take_action
    action(self, namespace, argument_values, option_string)
  File "/usr/lib/python3.6/argparse.py", line 1033, in __call__
    parser.print_help()
  File "/usr/lib/python3.6/argparse.py", line 2375, in print_help
    self._print_message(self.format_help(), file)
  File "/usr/lib/python3.6/argparse.py", line 2352, in format_help
    formatter.add_arguments(action_group._group_actions)
  File "/usr/lib/python3.6/argparse.py", line 285, in add_arguments
    self.add_argument(action)
  File "/usr/lib/python3.6/argparse.py", line 270, in add_argument
    invocations = [get_invocation(action)]
  File "/usr/lib/python3.6/argparse.py", line 566, in _format_action_invocation
    default = self._get_default_metavar_for_optional(action)
  File "/usr/lib/python3.6/argparse.py", line 698, in _get_default_metavar_for_optional
    return action.type.__name__
AttributeError: 'NoneType' object has no attribute '__name__'
```

I'm debugging why, but maybe those 2 'S' aren't really worth it..

Reply:
it works with type=str, metavar='SKIP_KEY' and adding type and metavar to --only as well.